geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStore.scala

```diff
@@ -647,6 +638,51 @@ class AccumuloDataStore(val connector: Connector,
     metadata.read(featureName, SCHEMA_KEY).getOrElse(EMPTY_STRING)
 
   /**
+   * Updates the index schema format - WARNING don't use this unless you know what you're doing.
+   * @param sft
+   * @param schema
+   */
+  def setIndexSchemaFmt(sft: String, schema: String) = {
+    metadata.insert(sft, SCHEMA_KEY, schema)
+    metadata.expireCache(sft)
+  }
+
+  /**
+   * Gets the internal geomesa version number for a given feature type
+   *
+   * @param sft
+   * @return
+   */
+  def geomesaVersion(sft: SimpleFeatureType): Int = geomesaVersion(sft.getTypeName)
```

Comment:
picky, and won't affect functionality, but could this be getGeomesaVersion() instead?